### PR TITLE
Addressing decryption plugin auditing findings.

### PR DESCRIPTION
**Pull request by stchrysa**

### CODE REVIEW

**Review by bolekk — APPROVED** [review]
LGTM!

File: go/ocr2/decryptionplugin/decryption.go [hunk 1]
```diff
@@ -79,7 +79,16 @@ func (dp *decryptionPlugin) Query(ctx context.Context, ts types.ReportTimestamp)
 	)
 
 	queryProto := Query{}
+	ciphertextIDs := make(map[string]bool)
 	for _, request := range decryptionRequests {
+		if _, ok := ciphertextIDs[string(request.CiphertextId)]; ok {
```
Comment (pszal) [inline go/ocr2/decryptionplugin/decryption.go:84]: no need to assign `ok` (the same in other places)

**Review by pszal — APPROVED** [review]

File: go/ocr2/decryptionplugin/decryption.go (hunk 1 above)
Comment (stchrysa) [reply go/ocr2/decryptionplugin/decryption.go:84]: Not sure I understand.

Comment (pszal) [reply go/ocr2/decryptionplugin/decryption.go:84]: I meant just to `if ciphertextIDs[string(request.CiphertextId)] {...`

**Pull request merged**